seeds/trials.js

```diff
@@ -479,6 +484,55 @@ exports.seed = (knex) => {
     },
   ];
 
+  const riskOfBiasCriterias = [
+    {
+      id: '756b06ca-8414-11e6-a6fa-e4b3181a2c8c',
+      name: '3.1',
+      created_at: new Date('2016-01-02'),
+      updated_at: new Date('2016-02-01'),
+    },
+    {
+      id: '7036b016-841c-11e6-a6fa-e4b3181a2c8c',
+      name: '3.2',
```

Comment:
Change the `name` from numbers to actual human-readable names (check `processors/cochrane/extractors.py#extract_review_results` on https://github.com/opentrials/processors/pull/59/files for example of names)